.github/workflows/pr-agent.yml

```diff
@@ -0,0 +1,14 @@
+on:
+  pull_request:
+  issue_comment:
+jobs:
+  pr_agent_job:
+    runs-on: ubuntu-latest
+    name: Run pr agent on every pull request, respond to user comments
+    steps:
+      - name: PR Agent action step
+        id: pragent
+        uses: Codium-ai/pr-agent@main
+        env:
+          OPENAI_KEY: ${{ secrets.OPENAI_API_KEY }}
+          GITHUB_TOKEN: ${{ secrets.TOKEN_GITHUB }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The workflow file seems to be correctly set up. It triggers on pull requests and issue comments, runs on an Ubuntu environment, and uses the `Codium-ai/pr-agent` GitHub Action. However, it's important to ensure that the secrets `OPENAI_API_KEY` and `TOKEN_GITHUB` are properly set in the repository settings. If they're not set or incorrectly set, the action will fail to run.

One more thing to note is that the action version is set to `@main`. This means it will always use the latest code from the main branch of the `Codium-ai/pr-agent` repository. While this ensures you have the latest features and bug fixes, it could potentially introduce breaking changes. Consider using a specific version of the action instead.

```diff
-        uses: Codium-ai/pr-agent@main
+        uses: Codium-ai/pr-agent@v1.0.0
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->